indico/web/templates/forms/protection_message_widget.html

```diff
@@ -0,0 +1,30 @@
+{% extends 'forms/base_widget.html' %}
+
+{% block html %}
+    {{ field._value()|safe }}
+{% endblock %}
+
+{% block javascript %}
+    <script>
+        (function() {
+            'use strict';
+
+            $('[name=protection_mode]').on('change', function() {
+                var $this = $(this);
+                var isProtected = $this.val() === 'protected' || ($this.val() === 'inheriting'
+                        && {{ field.get_form().object.protection_parent.is_protected | tojson }});
+
+                if (this.checked) {
+                    $('#form-group-protected .message-only').hide();
+                    $('#form-group-protected .' + $this.val() + '-protection-message').show();
+
+                    toggleAclField($('#protection-form [name=acl]'), !isProtected);
+                }
+            });
+
+            _.defer(function() {
+                $('[name=protection_mode]').trigger('change');
```

Comment:
This selector is way too generic. Selectors in form widgets ALWAYS need to use the ID of the field to avoid affecting anything outside.